docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/nft/get-nft-metadata.md

```diff
@@ -0,0 +1,153 @@
+---
+description: How to get NFT metadata on L1
+image: /img/logo/WASP_logo_dark.png
+tags:
+  - NFT
+  - EVM
+  - how-to
+---
+import GetNftMetadata from '../../../_partials/how-tos/token/_get-nft-metadata.md';
+
+# Get NFT Metadata
+
+This guide explains how to utilize the `getIRC27NFTData` function within a smart contract to fetch information about a specific IRC27 NFT on the IOTA Network. The function resides in the ISCSandbox contract and returns details pertaining to the requested NFT.
```

Comment:
```suggestion
This guide explains how to use the [`getIRC27NFTData`](../../../reference/magic-contract/ISCSandbox/#getirc27nftdata) function within a smart contract to fetch information about a specific IRC27 NFT on the IOTA Network. The function resides in the ISCSandbox contract and returns details pertaining to the requested NFT.
```

We could also cut the last sentence if we add the link to the refs.

Reply:
I have made the necessary updates based on your suggestions and also removed the last sentence as requested.

Reply:
getIRC27NFTData url was broken, fixed it with this url: https://github.com/iotaledger/wasp/blob/develop/packages/vm/core/evm/iscmagic/ISCSandbox.sol#L118

Reply:
I will fix it. But we need the updated reference docs first 👍